### DISCUSSION TITLE
New feature implementation

### DISCUSSION THREAD

**Discussion by MarkusPfundstein** [bbc/bmx#18] [General]
Hi Philip,

I am going to use BMX for a client project. We are going to implement the wrapping of some new formats that are not yet supported. I am currently studying the library and how it works and make good progress but I have some questions, especially regarding best practices, pitfalls etc.
Would there be a possibility to get in touch? Is there a discord or something?

Our intent is of course to contribute all our changes back to BMX. Please let me know :-)

Greetings
Markus

**Comment by philipnbbc** [comment]
Hi Markus, asking questions in here is the default way to get in touch, but if you prefer to use email for some questions then that is ok as well. The email address associated with the commits should work in that case.

**Reply by MarkusPfundstein** [reply]
Thanks Philip. Perfect. Currently working my way through the code. I will post here once I have a specific question that can be discussed in the public domain (we will open source later but until then, I can't disclose any specifics). Otherwise, I'll write you an email
